src/figuresClasses.ts

```diff
@@ -2,10 +1,97 @@
+export enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+export enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
```

Comment:
[CODE STYLE] - Enum constant names usually start with capital letter. Please revise the `Color` enum to follow this convention, for example, `Red` should be `RED`, `Green` should be `GREEN`, and `Blue` should be `BLUE`.